view/admin/components/common/Dropdown.js

```diff
@@ -179,7 +179,7 @@ class Dropdown extends React.Component {
     let outputUL = '';
     // if options exist, load them
     if (data.length) {
-      outputUL = <ul className="slds-dropdown__list" role="menu">
+      outputUL = <ul className="slds-lookup__list" role="menu">
```

Comment:
This css class is essential to have the two column effect. Otherwise the icon, text, and icon on the right all float to the left.
<img width="430" alt="screen shot 2017-02-10 at 9 51 46 am" src="https://cloud.githubusercontent.com/assets/17040750/22845949/02f7f426-ef9a-11e6-8086-f3d5f937da62.png">